crates/packager/src/config/mod.rs

```diff
@@ -1653,6 +1653,37 @@ impl Config {
             .map(|b| b.path.file_stem().unwrap().to_string_lossy().into_owned())
             .ok_or_else(|| crate::Error::MainBinaryNotFound)
     }
+
+    /// Returns all icons path.
+    pub fn icons(&self) -> crate::Result<Option<Vec<PathBuf>>> {
+        
+        let Some(patterns) = &self.icons else {
+            return Ok(None);
+        };
+
+        let mut paths = Vec::new();
+
+        for pattern in patterns {
+            match glob::glob(&pattern) {
+                Ok(icon_paths) => {
+                    for icon_path in icon_paths {
+                        match icon_path {
+                            Ok(path) => paths.push(path),
+                            Err(_e) => {
+                                //return e.into()
+                                panic!()
+                            },
+                        }
+                    }
+                },
+                Err(_e) =>  {
+                    //return e.into()
+                    panic!()
+                },
+            }
+        }
+        Ok(Some(paths))
```

Comment:
I believe this could be simplified to 
```suggestion
        let Some(patterns) = &self.icons else {
            return Ok(None);
        };

        let mut paths = Vec::new();

        for pattern in patterns {
            for icon_path in glob::glob(pattern)? {
                paths.push(icon_path?);
            }
        }

        Ok(Some(paths))
```